frr-metrics/collector/bgp.go

```diff
@@ -0,0 +1,140 @@
+package collector
+
+import (
+	"encoding/json"
+	"os/exec"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const bgpConnected = "Established"
+
+var (
+	bgpNamespace = "metallb"
+	bgpSubsystem = "bgp"
+
+	sessionUpDesc = prometheus.NewDesc(
```

Comment:
This seems to be lower level than using CounterVec / GaugeVec as the current implementation does.
Would it make sense to have the content of https://github.com/metallb/metallb/blob/2683861f566bf01fb3111cf37d6241f87bfca1dd/internal/bgp/stats.go#L5 exported as a package and consumable from here?

The benefit would be having one single place describing the metrics.